### DISCUSSION TITLE
二月更新

### DISCUSSION THREAD

**Discussion by Richasy** [Clean-Reader/CleanReader.Desktop#68] [Announcements]
## 🚀 功能

- 更新设置控件 @Richasy (#67)

## 🐛 Bug 修复

- 修复拆分TXT时最后一章缺失的问题 @Richasy (#66)
- 修复Ambie关闭后仍自动播放的问题 @Richasy (#65)

## 🧰 管理与维护

- 更新依赖引用 @Richasy (#57)

<hr /><em>This discussion was created from the release <a href='https://github.com/Clean-Reader/CleanReader.Desktop/releases/tag/v3.2206.2'>二月更新</a>.</em>

**Comment by LJLlw** [comment]
证书过期了😭
